provisioning/entrypoint.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+. $ASDF_DIR/asdf.sh
+
+exec /usr/local/bin/jenkins-agent "$@"
```

Comment:
@dduportal I didn't found where this file is defined in the original docker-agent image.
Is it supposed to be the `agent.jar` renamed `jenkins-agent`?

Reply:
as far as I can tell, it is defined in the inbound-agent's parent image: https://github.com/jenkinsci/docker-agent/blob/master/11/bullseye/Dockerfile#L50-L60.

Reading the link I provided, I know understand where does the `/usr/share/jenkins` comes from. Which makes sense somehow: this jar file is not a binary, but a file passed to the java command which starts the agent process, where `java` is the binary.

> renamed `jenkins-agent` without extension?

This one comes from the inbound-agent itself: https://github.com/jenkinsci/docker-inbound-agent/blob/master/11/debian/Dockerfile#L32-L34 . It looks to me that this is the entrypoint script.

Reply:
> This one comes from the inbound-agent itself: [jenkinsci/docker-inbound-agent@master/11/debian/Dockerfile#L32-L34](https://github.com/jenkinsci/docker-inbound-agent/blob/master/11/debian/Dockerfile?rgh-link-date=2022-07-22T17%3A43%3A52Z#L32-L34) .

I still don't understand where the `COPY ../../jenkins-agent` comes from or what is it. Any idea?